TESTING.asciidoc

```diff
@@ -537,6 +537,16 @@ ES_JAVA_OPTS="-Xdebug -Xrunjdwp:server=y,transport=dt_socket,address=4000,suspen
 
 Read your IDE documentation for how to attach a debugger to a JVM process.
 
+== Running unit tests from IntelliJ
+
+If you try to run the unit tests from IntelliJ, you might suffer from a Jar Hell issue.
+
+To fix it, add `idea.no.launcher=true` to your `idea.properties` and restart IntelliJ:
+
+---------------------------------------------------------------------------
+echo idea.no.launcher=true >> ~/Library/Preferences/IntelliJIdea2017.1/idea.properties
```

Comment:
@javanna Apparently on 2017.2 I had to add it to `idea.properties` although it was there with `-Didea.no.launcher=true` in `idea.vmoptions`.

@jasontedor fair point. Updated but with Custom properties instead.

LMK guys.